Tests/Io/PrintfTest.cpp

```diff
@@ -0,0 +1,19 @@
+#include <Printf.hpp>
+
+#include <random>
```

Comment:
You should checkout [`hsd::random`](https://github.com/HackyTeam/HackySTL/blob/dev/Tests/Random/RandTest.cpp)

Reply:
I know, but `hsd` doesn't have `random_device`